.github/workflows/linux-ci.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       run: sudo apt-get install pkg-config gcc gcovr gtk-doc-tools libfreetype6-dev libglib2.0-dev libcairo2-dev libicu-dev libgraphite2-dev python3 python3-setuptools ninja-build gobject-introspection libgirepository1.0-dev
     - run: sudo pip3 install fonttools meson==0.56.0
     - name: run
-      run: meson build -Db_coverage=true --auto-features=enabled -Dgraphite=enabled -Dchafa=disabled -Dragel_subproject=true -Doptimization=2
+      run: meson build -Db_coverage=true --auto-features=enabled -Dgraphite=enabled -Dchafa=disabled -Dragel=disabled -Doptimization=2
```

Comment:
We no longer test the ragel subproject on ci builds, or am I missing something?

Reply:
The project options specify `wrap_mode=nofallback`